### PR TITLE
Code changes..

### PR DESCRIPTION
**Pull request by NatarajBTI**
code changes..

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (daniel-butler-irl) [inline .project:1]: this file should not be checked into source

File: variables.tf.bak [hunk 2]
```diff
@@ -0,0 +1,116 @@
+##############################################################################
```
Comment (daniel-butler-irl) [inline variables.tf.bak:1]: this file should be removed, if you need it back you can look at the history

**Review by daniel-butler-irl — CHANGES_REQUESTED** [review]
two minor changes to keep the repo clean

**Comment by NatarajBTI** [comment]
@daniel-butler-irl - I removed those 2 files which have got checked-in by mistake..sorry for that..please review now..

**Review by daniel-butler-irl — APPROVED** [review]

**Pull request merged**